staking/programs/staking/src/state/positions.rs

```diff
@@ -33,7 +33,8 @@ impl PositionData {
     pub const LEN: usize = 8 + 32 + MAX_POSITIONS * POSITION_BUFFER_SIZE;
 }
 
-#[cfg(test)]
+// TODO: fix this
+// #[cfg(test)]
```

Comment:
compiler gives and error

Reply:
tbh don't mind getting rid of this. Not a big fan of these conditional implementations if we are in test mode anyway